modules/ROOT/pages/cloudhub.adoc

```diff
@@ -105,6 +86,8 @@ Through various tools, Runtime Manager allows you to triage problems, view logs,
 * xref:cloudhub-architecture.adoc[CloudHub Architecture]
 * xref:cloudhub-fabric.adoc[CloudHub Fabric]
 * xref:cloudhub-networking-guide.adoc[CloudHub Networking Guide]
+* xref:managing-deployed-applications.adoc[Managing Deployed Applications] 
+* xref:managing-applications-on-cloudhub.adoc[Managing Applications on CloudHub] 
 * xref:penetration-testing-policies.adoc[Penetration Testing Policies]
 * https://www.mulesoft.com/lp/whitepaper/saas/cloud-security[Cloud Security and Compliance Whitepaper]
-* http://www.mulesoft.com/cloudhub/ipaas-cloud-based-integration-demand[things you can do with CloudHub].
\ No newline at end of file
+* http://www.mulesoft.com/cloudhub/ipaas-cloud-based-integration-demand[things you can do with CloudHub].
```

Comment:
Just curious why the title here begins with a lower case 't'. Is it not a title? Ignore if not applicable.